script.js

```diff
@@ -1,12 +1,44 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let img;
+let backgroundImg;
+function preload() {
+  img = loadImage('./images/turtle.svg');
+  backgroundImg = loadImage('./images/beach.svg')
+}
 
 function setup() {
-  createCanvas(windowWidth, windowHeight);
-    background(255);
+  createCanvas(displayWidth, displayHeight);
+  backgroundImg.resize(windowHeight,windowWidth - 200)
+  background(backgroundImg)
+  noStroke()
+ 
+  
 }
 
 function draw() {
-  noStroke();
-  fill(random(colorlist));
-  ellipse(mouseX, mouseY, 25, 25);
+  img.resize(200,200);
+  cursor(image(img, mouseX-20,mouseY-20))
+  frameRate(3)
+  
+  fill('#C6B6A8')
+  rotate(.15)
```

Comment:
love the use of rotation here! Would be good to pair this with a comment to explain what aesthetic you are trying to accomplish with the rotation.